bin/run

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Figure out where we are.
+FWDIR="$(cd `dirname $0`; pwd)"
+
+DATE=`date +%Y%m%d.%H%M%S`
+
+export HADOOP_HOME=/usr
+export KEYSTONE_DIR=$FWDIR/../../keystone-test
+export KEYSTONE_VERSION=$1 #TODO: Argument validation.
+export JAVA_HOME=/usr/java/jdk1.8.0_65
+
+export KEYSTONE_MEM=${KEYSTONE_MEM:-20g}
+export HDFS_PATH=/user/`whoami`
+export LOCAL_PATH=~/tmpdata/
+
+if [ ! -d LOCAL_PATH ]; then
+  mkdir -p $LOCAL_PATH
+fi
+
+if [ ! -d logs ]; then
+  mkdir -p logs
+fi
+
+TEST_NAME=all #TODO: Make this a parameter.
+LOG_FILE=logs/$TEST_NAME.$DATE.log
+
```

Comment:
Should we add a step to do `sbt/sbt assembly` in the `keystone-integration-tests` package as well ? Right now that needs to be done before this right ?